vali_objects/utils/auto_sync.py

```diff
@@ -581,6 +587,118 @@ def sync_positions_with_cooldown(self, auto_sync_enabled:bool):
 
         self.last_signal_sync_time_ms = TimeUtil.now_in_millis()
 
+    def add_checkpoint(self, received_checkpoint: json, total_checkpoints):
```

Comment:
going to remove these from PositionSyncer class and move to newly created P2PSyncer class